src/main/java/mekanism/common/tile/factory/TileEntityFactory.java

```diff
@@ -760,11 +760,9 @@ public ErrorTracker(Map<RecipeError, Boolean> errorTypes, int processes) {
         }
 
         private void track(MekanismContainer container) {
-            for (boolean[] processTrackedErrors : trackedErrors) {
-                //Note: We don't need a special syncable boolean for two depth arrays
-                // as long as our arrays are in the same order on client and server
-                for (int j = 0; j < processTrackedErrors.length; j++) {
-                    container.track(SyncableBoolean.create(processTrackedErrors, j));
+            for (int i = 0; i < trackedErrors.length; i++) {
+                for (int j = 0; j < trackedErrors[i].length; j++) {
+                    container.track(SyncableBoolean.create(trackedErrors, i, j));
                 }
             }
         }
```

Comment:
Is there a reason this method still exists instead of being replaced with a call to `container.trackArray(trackedErrors)`